### PR TITLE
Refactor

### PR DESCRIPTION
**Pull request by BoBer78**

### CODE REVIEW

**Review by cszsol — APPROVED** [review]
LGTM
I don't have any issues so I will approve it, but lets wait for another review.

File: CHANGELOG.md
Comment (WonderPG) [inline CHANGELOG.md:14]: Not needed anymore with release please I guess ?

**Review by WonderPG — COMMENTED** [review]
LGTM I guess we will have another round of refactoring once integrating our logic anyway.

**Review by WonderPG — APPROVED** [review]

**Pull request merged**
